php/config-spec.php

```diff
@@ -8,8 +8,8 @@
 	),
 
 	'ssh' => array(
-		'runtime' => '=<ssh>',
-		'file' => '<ssh>',
+		'runtime' => '=[<user>]@<host>[:<port>][<path>]',
+		'file' => '[<user>]@<host>[:<port>][<path>]',
```

Comment:
This should be `[<user>@]`, because the `@` isn't necessary when the user isn't included.